governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+
+#### 12.2 Finding of "substantial threat"
+
+A substantial threat to the functioning of Maker Governance is one that extends beyond causing mere inconvenience, annoyance or dissatisfaction. To be "substantial," the threat must be of both considerable *imminence* and *magnitude*. Therefore, a threat that is of considerable magnitude but is not likely to present in the near-term could not be characterized as "substantial."
+
+To determine whether a substantial threat exists to the functioning of Maker Governance such that GOV 12 may be triggered, the Governance Facilitator must engage in a balancing inquiry. This inquiry explicitly weighs two interests: the imminence and magnitude of the threat, against the need to uphold a decentralized governance equilibrium pursuant to customary governance protocols. 
+
+In undertaking this balancing inquiry, the Governance Facilitator is required to consider alternative measures to resolve the threat, which measures comport with Maker's customary governance protocols.
+
+#### 12.3 General Protocol
+
+Governance Facilitators may initiate a GOV 12 Governance Poll only if *all* of the following conditions are met:
+
+- The Governance Facilitator must signal their intention to initiate a GOV 12 Governance Poll in a Forum post. The Forum post must document, in reasonable detail, the Facilitator's 1) analysis and reasoning in undertaking the balancing inquiry and 2) the proposed revisions to the Atlas/Scope Artifacts. The post must contain a formal conflict-of-interest declaration. 
+- Two endorsements of the Governance Facilitator's post are required. One Facilitator *and* one Advisory Council Member of the impacted Scope(s) must post on the Forum their explicit endorsement. The endorsers must substantively explain their support of the Governance Facilitator's proposal. The endorsers must also include a formal conflict-of-interest declaration. 
+- In order to qualify as a GOV 12 endorser, the Facilitator and Advisory Council Member must be currently active in their respective roles and otherwise be in good standing. If a Facilitator has received a Formal Warning per [GOV 4.2](https://mips.makerdao.com/mips/details/MIP113#4-2-alignment-conserver-formal-warnings), they cannot qualify as a GOV 12 endorser.
+- Once a Facilitator/Advisory Council Member has acted in the capacity of a GOV 12 endorser, they are prohibited from acting in such a capacity again until two full calendar months have passed. 
+
+#### 12.4 Exceptions to the General Protocol
+
+##### 12.4.1 Clear Mistakes
+
+Excepted from the General Protocol are *clear* mistakes, such as typos or inconsistencies, in the text of the Atlas/Scope Artifacts that obstruct the critical functioning of Maker Governance. 
```

Comment:
'critical functioning' eh? :) Might be better to refer back to the concepts of imminence and magnitude?